src/Summary.jsx

```diff
@@ -36,14 +37,15 @@ class Summary extends React.Component {
 
   componentDidMount() {
     const { activeBlock } = this.props;
+    const request = activeBlock ? RestAPI.requestGeometryByArea(activeBlock.id) : RestAPI.requestGeometryByArea('LLA 0970');
```

Comment:
It would be better to have an if before:
`const blockId = (activeBlock && activeBlock.id) ? activeBlock.id : 'LLA 0970'`
That way the call to `RestAPI.requestGeometryByArea(blockId)` is clean and doesn't need to be changed in the future